shared/p2p/service.go

```diff
@@ -101,6 +101,7 @@ func checkAvailablePort(port int) bool {
 		return false
 	}
 
+	// #nosec G104
```

Comment:
Why are we ignoring this error? Please handle the error.

Reply:
Alright handled the error